jgrapht-core/src/main/java/org/jgrapht/traverse/CrossComponentIterator.java

```diff
@@ -55,12 +56,18 @@
     /**
      * Iterator which provides start vertices for cross-component iteration.
      */
+    private Iterator<V> entireGraphVertexIterator = null;
```

Comment:
What do we want to happen if the user changes the value of crossComponentTraversal?